.travis.yml

```diff
@@ -56,9 +56,7 @@ jobs:
             chmod 600 $TRAVIS_BUILD_DIR/.deploy_rsa &&
             eval $(ssh-agent -s) && ssh-add $TRAVIS_BUILD_DIR/.deploy_rsa &&
             scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null /tmp/oqdata.zip travis@ci.openquake.org:/var/www/artifacts.openquake.org/travis/oqdata-${BRANCH}.zip &&
-            oq workers stop &&
-            oq reset --yes &&
```

Comment:
Why we are not testing `oq reset` anymore?

Reply:
Because I forgot to restore the removed lines. Fixed here: https://github.com/gem/oq-engine/commit/e1e21031857d9dec33fa8548655d054d92e4dc6a